mycroft/client/wifisetup/main.py

```diff
@@ -218,12 +402,20 @@ def connect(self, event=None):
                 connected = self.get_connected(ssid)
                 if connected:
                     wpa(self.iface, 'save_config')
-                    # ConfigurationManager.set(self.NAME, 'setup', False, True)
 
             self.client.emit(Message("mycroft.wifi.connected",
                                      {'connected': connected}))
             LOG.info("Connection status for %s = %s" % (ssid, connected))
 
+            if connected:
+                if True:
```

Comment:
This is a placeholder for code that will determine if pairing of the unit has already occurred.  When it hasn't, the pairing code will be displayed at this point to continue walking the user through setting up their device.